plugins/apache_http.yaml

```diff
@@ -107,4 +130,57 @@ pipeline:
           - trace7
           - trace8
     output: {{ .output }}
-  #{{ end }}
+  # {{ end }}
+
+  # {{ if eq $log_format "observiq" }}
+  - id: access_json_parser
+    type: json_parser
+    timestamp:
+      parse_from: timestamp
+      layout: '%Y-%m-%dT%H:%M:%S.%s%z'
+    severity:
+      parse_from: status
+      preset: none
+      preserve_to: status
+      mapping:
+        info: 2xx
+        notice: 3xx
+        warning: 4xx
+        error: 5xx
+    output: access_protocol_parser
+
+  - id: access_protocol_parser
+    type: regex_parser
+    parse_from: $record.request.protocol
+    parse_to: $record.request
+    regex: '(?P<protocol>[^/]*)/(?P<protocol_version>.*)'
+    output: {{ .output }}
+
+  - id: error_json_parser
+    type: json_parser
+    timestamp:
+      parse_from: time
+      layout: '%Y-%m-%d %H:%M:%S.%s'
+    severity:
+      parse_from: log_level
+      mapping:
+        critical: crit
```

Comment:
Looking at this, should we also include info? https://stackify.com/apache-error-log-explained/

Reply:
By default mapping includes info. We map the others as they are different than the default mappings.